ApplicationLibCode/ProjectDataModel/Surfaces/RimSurface.cpp

```diff
@@ -221,6 +224,13 @@ RigSurface* RimSurface::surfaceData()
     return m_surfaceData.p();
 }
 
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+void RimSurface::loadSurfaceDataForTimeStep( int timeStep )
+{
```

Comment:
Add comment on what to do if time dependent data is available

Reply:
Fixed.